FWCore/Framework/src/DroppedDataProductResolver.h

```diff
@@ -13,18 +13,16 @@ namespace edm {
   class DroppedDataProductResolver : public ProductResolverBase {
   public:
     DroppedDataProductResolver(std::shared_ptr<ProductDescription const> bd)
-        : ProductResolverBase(), m_provenance(std::move(bd), {}) {}
+        : ProductResolverBase(), m_provenance(bd, {}) {}
```

Comment:
I see this change was from `clang-tidy`.

Reply:
yes